sdk/tables/test-resources-post.ps1

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script is used to set up CORS rules for tables after storage account creation by New-TestResources.ps1
+# There are no documented approaches to specifying CORS rules using ARM, this is a workaround until
+# support for setting CORS rules is added to ARM for Tables
+
+# It is invoked by the https://github.com/Azure/azure-sdk-for-js/blob/master/eng/New-TestResources.ps1
+# script after the ARM template, defined in https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/test-resources.json,
+# is finished being deployed. The ARM template is responsible for creating the Storage accounts needed for live tests.
+
+param (
+  [hashtable] $DeploymentOutputs
+)
+
+$storageAccountName = $DeploymentOutputs['ACCOUNT_NAME']
+$context = New-AzStorageContext -StorageAccountName $storageAccountName
```

Comment:
Can you try to pass ACCOUNT_KEY also to context like in following command. https://github.com/Azure/azure-sdk-for-js/blob/49f21adf3fa4b3359b9d759c85a29f63d9b6583f/sdk/storage/test-resources-post.ps1#L55

Reply:
Thanks! it worked 😄